src/MqttSettings.cpp

```diff
@@ -228,6 +290,23 @@ bool MqttSettingsClass::getConnected()
     return mqttClient->connected();
 }
 
+String MqttSettingsClass::getVictronPortalId()
+{
+    return VictronPortalId;
+}
+
+String MqttSettingsClass::getVictronDeviceInstance(String hoyserial)
+{
+    if (VictronDeviceInstance.find(hoyserial)!=VictronDeviceInstance.end()) {
+        return VictronDeviceInstance[hoyserial];
+    } else {
+        Serial.print(F("No Victron deviceInstance found for inverter: "));
+        Serial.println(hoyserial);
+        String ret = hoyserial + "NOdevInstance";
+        return ret;
+    }
+}
+
```

Comment:
Doesn't make it more sense to move this methods to the MqttVictronPublishing class?

Reply:
That is, because of my poor understanding of c-classes. I learned only traditional C. 
In the MqttSettings is the handling of the Mqtt messages getting in, line 122f. Then i have the values for portalId and deviceinstance in this class and from the other class i get this values with this methods.
Is there a better way to do this?